### PR TITLE
Readme2

### PR DESCRIPTION
**Pull request by nicolebussola**
- add quickstart

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
 
 The aim of this project is to provide a tool for WSI processing in a reproducible environment to support clinical and scientific research. HistoLab is designed to handle WSIs, automatically detect the tissue, and retrieve informative tiles, and it can thus be integrated in a deep learning pipeline.
 
-## Getting Started
+## Getting Started 
 
-### Prerequisites
+### Prerequisites 
 
 HistoLab has only one sistem-wide dependency: OpenSlide.
```
Comment (alessiamarcolini) [inline README.md:44]: the L is lower-case

File: README.md [hunk 2]
```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
...
+```python
+breast_slide.show() 
+heart_slide.show()
+```
+
+![caption](images/thumbnails.png)
+
+### Tiles extraction
+
+Now that your slide object is defined, you can automatically extract the tiles. ```RandomTiler``` method crops random tiles from the slide.
```
Comment (alessiamarcolini) [inline README.md:156]: `RandomTiler` is not a method. You could say: A `RandomTiler` object crops...

(note the A before)

Comment (alessiamarcolini) [inline README.md:156]: ..your `Slide` object is defined...

File: README.md [hunk 3]
```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
...
+You need to specify the size you want your tiles, the number of tiles to crop, and the level of magnification. If ```check_tissue``` is True, the exracted tiles are taken by default from the **biggest tissue region detected** in the slide, and the tiles are saved only if they have at least 80% of tissue inside.
+
+
+```python
+from histolab.tiler import RandomTiler
+
+random_tiles_extractor = RandomTiler(tile_size=(512,512), n_tiles=6, level=2, seed=42, check_tissue=True, prefix='processed/breast_slide/')
```
Comment (alessiamarcolini) [inline README.md:163]: line too long

File: README.md [hunk 4]
```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
...
+    	 Tile 5 saved: processed/breast_slide/tile_5_level2_78649-3569-86842-11763.png
+    	 Tile 6 saved: processed/breast_slide/tile_6_level2_81994-6753-90187-14948.png
+    6 Random Tiles have been saved.
+
+
+![caption](images/breast.png)
+
+
+```python
+random_tiles_extractor = RandomTiler(tile_size=(512,512), n_tiles=6, level=0, seed=42, check_tissue=True, prefix='processed/heart_slide/')
```
Comment (alessiamarcolini) [inline README.md:181]: line too long

File: README.md [hunk 5]
```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
...
+    	 Tile 4 saved: processed/heart_slide/tile_4_level0_11472-26400-11984-26912.png
+    	 Tile 5 saved: processed/heart_slide/tile_5_level0_13489-42680-14001-43192.png
+    	 Tile 6 saved: processed/heart_slide/tile_6_level0_13281-33895-13793-34407.png
+    6 Random Tiles have been saved.
+
+
+![caption](images/heart.png)
+
+
+```python
```
Comment (alessiamarcolini) [inline README.md:198]: empty box

File: README.md [hunk 6]
```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
...
+```
+
+
+```python
+from PIL import Image
+from IPython.display import Image as show_image
+```
+
+The module ```data``` contains some WSIs retrieved from the [TCGA data portal](https://portal.gdc.cancer.gov/) and [OpenSlide test data](http://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/), to be used as examples. The available images are H&E tissue sections stored in .svs format:
+* ```aorta_tissue```: Aorta tissue slide (OpenSlide)
```
Comment (alessiamarcolini) [inline README.md:68]: for the quick start it's not really needed to have all of these explained. Let's keep them for a longer tutorial

Comment (nicolebussola) [reply README.md:68]: OK

File: README.md (hunk 4 above)
Comment (nicolebussola) [reply README.md:181]: How can I do?

Comment (alessiamarcolini) [reply README.md:181]: write it in a random py script and blackify it!

File: .gitignore [hunk 7]
```diff
@@ -27,13 +27,15 @@ eggs/
 lib/
 lib64/
 parts/
+processed/
```
Comment (ernestoarbitrio) [inline .gitignore:30]: i'd remove this from here

File: README.md [hunk 8]
```diff
@@ -35,47 +35,181 @@ Whole Slide Images (WSIs), namely the translation of tissue slides from glass to
...
+    	 Tile 4 saved: processed/breast_slide/tile_4_level2_78969-3953-87162-12147.png
+    	 Tile 5 saved: processed/breast_slide/tile_5_level2_78649-3569-86842-11763.png
+    	 Tile 6 saved: processed/breast_slide/tile_6_level2_81994-6753-90187-14948.png
+    6 Random Tiles have been saved.
+
+
+![breast 001](https://user-images.githubusercontent.com/31658006/84955724-0f1a9600-b0f8-11ea-92c9-3236dd16bca8.png)
+
+```python
+random_tiles_extractor = RandomTiler(tile_size=(512,512), n_tiles=6, level=0, seed=42, check_tissue=True, prefix='processed/heart_slide/')
```
Comment (alessiamarcolini) [inline README.md:177]: still too long

**Pull request merged**